packages/fuels-signers/src/provider.rs

```diff
@@ -33,6 +35,25 @@ pub struct TransactionCost {
     pub total_fee: u64,
 }
 
+#[derive(Debug)]
+// ANCHOR: time_parameters
+pub struct TimeParameters {
+    // The time to set on the first block
+    pub start_time: u64,
+    // The time interval between subsequent blocks
+    pub block_time_interval: u64,
```

Comment:
One thing that's not clear to me (from a user perspective) is the unit of time in this `block_time_interval`. Is it seconds, ms, ns? I think we should make it clear in the name of the property itself or at least in the comment.

Reply:
I've changed these fields to use `chrono::DateTime` and `chrono::Duration`. That should make it more transparent.